iib/workers/tasks/build.py

```diff
@@ -3,6 +3,7 @@
 import logging
 import os
 import re
+import sqlite3
```

Comment:
this should be probably added to requirements.txt

Reply:
+1 will do

Reply:
just realizing this is included in the standard library for python: https://stackoverflow.com/questions/19530974/how-can-i-add-the-sqlite3-module-to-python

Reply:
ah, cool